### ISSUE TITLE
Windows 10, ModuleNotFoundError: No module named 'nptdms', excel same. 

### ISSUE THREAD

**Issue by RAJUL555** [ebranlard/weio#21]
Hi, 

I can open .csv files, but can not excel or .tdms files. Says "no module". installed on Windows 10, pyDatView v0.3-7.  

edit: checked pyDatView config file:
     C:\Users\********\AppData\Roaming\pyDatView\pyDatView.json
weio data directory:     
     C:\Users\********\AppData\Roaming\weio
     
     weio folder is missing.

Best,

![image](https://user-images.githubusercontent.com/93572329/184112677-fff9add1-a7fb-4923-a2c9-a93d0708a21f.png)

**Comment by ebranlard** [comment]
Dear @RAJUL555,

Thank you for reporting this and for testing pyDatView. The dev version of 0.3 I've been through some rough times lately as I've updated the executable to python 3.9 and changed the directory structure. 

Your issue if mostly a `pyDatView` issue now (because from version 0.3 on, weio and pyDatView are slightly decoupled). 
If you don't mind, I've created an issue here, we can discuss your problem further:  
       https://github.com/ebranlard/pyDatView/issues/123

For `weio`, `nptdms` was an optional requirements. I hadn't used nptdms for a while and I can see that the interface of this library has changed. I've committed a fix in the latest dev branch (0b61a6275626ac861bf6b1fce4ba011df54b0d35 and 083b69e83c3c34b672b5bb65f22ab61c12825794) to update the reader and added the nptms library to the requirements. I've only used this format once. If you don't mind sending me a "minimal" file, with few time steps (say 10), and few groups/channels, so that the file is only a couple of KB, then I can add it to the unit test of `weio`, to avoid issues in the future.  


For pyDatView, we can discuss in the pyDatView issue.

**Comment by RAJUL555** [comment]
Dear Ebranlard,

[12_Merge_file_2022-08-16_12-52-00.zip](https://github.com/ebranlard/weio/files/9350082/12_Merge_file_2022-08-16_12-52-00.zip)

here is a  zipped "minimal file" for tests. Thanks.

Best.

**Comment by ebranlard** [comment]
Hi @RAJUL555 

Thanks a lot for that. I can see that you file would have not worked properly with the current reader. 
I got to learn a bit more about TDMS file. I've uploaded a test case and some corrections of the `tdms_file.py` scripts in [8ea2c4a](https://github.com/ebranlard/weio/commit/8ea2c4a44a9aeadc8f6c533080fd6a6c98c5999d). 

 Version v0.3-37 of pyDatView  (https://github.com/ebranlard/pyDatView/releases/tag/v0.3-dev) should be able to read the tdms files , provided you unzip the nptdms zip file as mentioned in https://github.com/ebranlard/pyDatView/issues/123
